stablehlo/tests/infer_stablehlo.mlir

```diff
@@ -330,6 +330,20 @@ func.func @case(%index : tensor<i32>, %branch_operand : tensor<2xf32>) {
 
 // -----
 
+func.func @sort(%input0: tensor<16x16xf32>, %input1: tensor<16x16xi32>) {
+  // CHECK: stablehlo.sort
+  %0:2 = "stablehlo.sort"(%input0, %input1) ({
+  ^bb0(%arg0: tensor<f32>, %arg1: tensor<f32>, %arg2: tensor<i32>, %arg3: tensor<i32>):
+    %7 = "stablehlo.compare"(%arg0, %arg1) {comparison_direction = #stablehlo<comparison_direction GT>} : (tensor<f32>, tensor<f32>) -> tensor<i1>
+    "stablehlo.return"(%7) : (tensor<i1>) -> ()
+  }) {dimension = 1 : i64, is_stable = true} : (tensor<16x16xf32>, tensor<16x16xi32>) -> (tensor<16x16xf32>, tensor<16x16xi32>)
+  // CHECK: (tensor<16x16xf32>) -> tensor<16x16xindex>
+  %1 = "hlo_test_infer.get_return_type_components"(%0) : (tensor<16x16xf32>) -> tensor<16x16xindex>
```

Comment:
Same comment as in the nearby PR for `stablehlo.while`. Why does `get_return_type_components` only call out the first element of the result?

Reply:
Fixed.